### DISCUSSION TITLE
reading a rosbag

### DISCUSSION THREAD

**Discussion by fbliman** [JuliaRobotics/Caesar.jl#920] [Q&A]
Has anyone recently tried to read a rosbag for then setting up the graph for a SLAM.

I am quite new with SLAM and Julia and I've been a little bit stucked with this since the documentation and the library [RobotOS.jl](https://github.com/jdlangs/RobotOS.jl) seems to be a bit out of date.

If anyone has an example of how can this be done it would help me a lot.

Thanks

**Comment by simenhu** [comment]
Same here, is there any development on this currently?
